plugins/node/instrumentation-undici/src/undici.ts

```diff
@@ -113,16 +106,6 @@ export class UndiciInstrumentation extends InstrumentationBase<UndiciInstrumenta
     this.subscribeToChannel('undici:request:error', this.onError.bind(this));
   }
 
-  override setConfig(config: UndiciInstrumentationConfig = {}): void {
```

Comment:
Note for reviewer: here I was mixing `UndiciInstrumentationConfig` with `InstrumentationConfig`. This side effect shouldn't be there